package.json

```
@@ -43,11 +43,17 @@
     "is-ci": "^3.0.0",
     "jest": "^27.1.0",
     "rollup": "^2.56.3",
-    "rollup-plugin-terser": "^7.0.2",
     "semver": "^7.3.5",
     "ts-jest": "^27.0.5",
     "tslib": "^2.3.1",
     "tslint": "^6.1.3",
     "typescript": "^4.3.5"
+  },
+  "exports": {
+    "production": "./dist/observable-membrane.prod.js",
+    "import": "./dist/observable-membrane.js"
+  },
+  "engines": {
+    "node": ">=12"
```

```suggestion
    "node": "^12.20.0 || ^14.13.1 || >=16.0.0"
```

Via https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c